sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/VectorIndexTest.java

```diff
@@ -326,12 +326,14 @@ private List<CosmosVectorIndexSpec> populateVectorIndexes() {
         cosmosVectorIndexSpec2.setPath("/vector2");
         cosmosVectorIndexSpec2.setType(CosmosVectorIndexType.QUANTIZED_FLAT.toString());
         cosmosVectorIndexSpec2.setQuantizationSizeInBytes(2);
+        cosmosVectorIndexSpec2.setVectorIndexShardKeys(Arrays.asList("/zipCode"));
```

Comment:
where is this being tested?

Reply:
Added assertion check in `validateVectorIndexes` method.